main.py

```diff
@@ -12,9 +12,10 @@
 from steganography.user_interface import Steganography
 
 # Fonts
+pygame.init()
 font = pygame.font.Font(None, 36)
 
-PROBLEMS = [PasswordCracker(), NetworkSniffer(), Steganography()]
+PROBLEMS = [PasswordCracker(), NetworkSniffer()]#, Steganography()]
```

Comment:
Is there any reason you omitted the steganography challenge?

Reply:
It is not working correctly on my side. I didn't have time to check exactly what the problem is, but I'll look into it tonight.